idprovider/local/cache.go

```diff
@@ -104,13 +104,13 @@ func (c *contactsCache) Write(alias string, p perun.Peer) error {
 	return nil
 }
 
-// Delete deletes the peer from contacts cache.
+// Delete deletes the peer from idProvider cache.
```

Comment:
```suggestion
// Delete deletes the peer from ID provider cache.
```

same as above 
Using ID provider as two words is more readable in the context of comments and documentation.

Reply:
[Done](https://github.com/hyperledger-labs/perun-node/pull/135/commits/c55f1f275ed66aae884d2380810cc6c5c06190e1)